engine/render/src/render/render_script.cpp

```diff
@@ -1199,7 +1234,7 @@ namespace dmRender
             lua_pop(L, 2);
         }
 
-        if (InsertCommand(i, Command(COMMAND_TYPE_SET_RENDER_TARGET, (uintptr_t)render_target, transient_buffer_types)))
+        if (InsertCommand(i, Command(COMMAND_TYPE_SET_RENDER_TARGET, (uintptr_t) dmGraphics::GetOpaqueHandle(render_target), transient_buffer_types)))
```

Comment:
Why do we need to convert from an asset handle to a raw handle, and then in the backend convert it back to an asset handle?

I would have thought that the asset handle was the handle to be used all the way?

Reply:
Yes, it's because we use uintptr_t for the operands in the render commands which are defined as uint32 in 32b systems. So the handle will be truncated. I think maybe for this iteration of the feature we could use 64b operand types in the render commands and skip the conversion

Reply:
Yeah `uint64_t` sounds like the way forward.